pkg/eventreason/eventreason.go

```diff
@@ -0,0 +1,52 @@
+/*
+Copyright 2020 The KEDA Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package eventreason
+
+const (
```

Comment:
This is something I'd like to discuss @ahmelsayed @zroubalik because when you are on the other side of the fence (like I am with https://github.com/tomkerkhove/k8s-event-bridge) it is super important that these are unique and clear.

I'd suggest that we always add a KEDA prefix or suffix, I don't know what you think?

For example `Ready` is very generic and makes sense in this context, but when processing events it's hard to determine what is ready based on the reason. For example what if it's `ScaledObjectIsReady` or `KEDAScaledObjectIsReady`